src/Compilers/CSharp/Portable/Parser/LanguageParser_InterpolatedString.cs

```diff
@@ -103,7 +103,7 @@ private ExpressionSyntax ParseInterpolatedStringToken()
                 var text = Substring(originalText, openQuoteIndex + 1, closeQuoteIndex - 1);
                 if (text.Length > 0)
                 {
-                    var token = MakeStringToken(text, text, isVerbatim, SyntaxKind.InterpolatedStringTextToken);
+                    var token = MakeInterpolatedStringTextToken(text, isVerbatim);
```

Comment:
in all cases the second argument matched the first.  The last argument was always passed the exact same value.